concepticondata/conceptlists/Wilson-2015-173.tsv

```diff
@@ -0,0 +1,174 @@
+Wilson-2015-173-10	10	BIG	grandis	grande	big	1	Adj.	24	4,31	0,04	478,09	1202	BIG
+Wilson-2015-173-11	11	BIRD	avis	ave	bird	1	Noun	15	2,15	0,03	28,53	937	BIRD
+Wilson-2015-173-12	12	BLACK	niger	negro	black	1	Adj.	15	1,91	0,03	145,15	163	BLACK
+Wilson-2015-173-13	13	BLOOD	sanguis	sangre	blood	2	Noun	15	2,19	0,04	95	946	BLOOD
+Wilson-2015-173-14	14	BONE	os	hueso	bone	1	Noun	13	1,34	0,03	26,32	1394	BONE
+Wilson-2015-173-15	15	CLOUD	nubes	nube	cloud	1	Noun	24	3,25	0,03	19,71	1489	CLOUD
+Wilson-2015-173-16	16	COLD	frigidus	frío	cold	1	Adj.	22	3,58	0,03	104,85	1287	COLD
+Wilson-2015-173-17	17	DAY (NOT NIGHT)	dies	día	day (not night)	1	Noun	10	1,04	0,03	1459,56	1225	DAY (NOT NIGHT)
+Wilson-2015-173-18	18	DIRTY	sordidus	sucio	dirty	4	Adj.	46	9,27	0,04	31,18	1230	DIRTY
+Wilson-2015-173-19	19	DOG	canis	perro	dog	4	Noun	15	1,79	0,03	128,82	2009	DOG
+Wilson-2015-173-20	20	DRY (SUBSTANCE)	siccus	seco	dry (substance)	0	Adj.	13	1,83	0,04	23,97	1398	DRY
+Wilson-2015-173-21	21	DUST	pulvis	polvo	dust	2	Noun	26	3,63	0,03	20,44	2	DUST
+Wilson-2015-173-22	22	EAR	auris	oreja	ear	4	Noun	6	0,88	0,05	101,91	1247	EAR
+Wilson-2015-173-23	23	EARTH (SOIL)	solum	suelo	earth (soil)	1	Noun	23	3,52	0,03	137,65	1228	EARTH (SOIL)
+Wilson-2015-173-24	24	EGG	ovum	huevo	egg	1	Noun	13	1,57	0,04	22,35	744	EGG
+Wilson-2015-173-25	25	EYE	oculus	ojo	eye	1	Noun	8	0,93	0,04	235,59	1248	EYE
+Wilson-2015-173-26	26	FAR	procul	lejos	far	4	Adj.	19	3,14	0,04	90,88	1406	FAR
+Wilson-2015-173-27	27	FAT	adeps	grasa	fat	4	Noun	31	4,89	0,03	14,71	323	FAT (ORGANIC SUBSTANCE)
+Wilson-2015-173-28	28	FATHER	pater	padre	father	1	Pronoun	9	2,3	0,06	499,12	1217	FATHER
+Wilson-2015-173-29	29	FEATHER	pluma	pluma	feather	1	Noun	15	2,35	0,04	18,68	1201	FEATHER
+Wilson-2015-173-30	30	FIRE	incendium	incendio	fire	1	Noun	15	1,75	0,03	50,59	221	FIRE
+Wilson-2015-173-31	31	FISH	piscis	pez	fish	1	Noun	11	1,45	0,04	18,97	227	FISH
+Wilson-2015-173-32	32	FIVE	quinque	cinco	five	2	Number	1	0,09	0,09	377,5	493	FIVE
+Wilson-2015-173-33	33	FLOWER	flos	flor	flower	1	Noun	18	2,29	0,03	109,56	239	FLOWER
+Wilson-2015-173-34	34	FOG	nebula	niebla	fog	1	Noun	32	4,97	0,04	5,15	249	FOG
+Wilson-2015-173-35	35	FOOT	pes	pie	foot	0	Noun	12	1,39	0,03	142,21	1301	FOOT
+Wilson-2015-173-36	36	FOUR	quattuor	cuatro	four	2	Number	2	0,38	0,13	406,32	1500	FOUR
+Wilson-2015-173-37	37	FRUIT	fructus	fruto	fruit	5	Noun	23	2,95	0,03	22,65	1507	FRUIT
+Wilson-2015-173-38	38	GOOD	bonus	buen	good	0	Adj.	25	3,04	0,03	2433,68	1035	GOOD
+Wilson-2015-173-39	39	GRASS	herba	hierba	grass	1	Noun	22	2,73	0,03	7,94	606	GRASS
+Wilson-2015-173-40	40	GREEN	viridis	verde	green	1	Adj.	19	2,46	0,03	58,82	1425	GREEN
+Wilson-2015-173-41	41	GUTS	intestinum	intestino	guts	1	Noun	35	6,89	0,04	5,44	1334	GUTS
+Wilson-2015-173-42	42	HAIR	pilus	pelo	hair	1	Noun	24	3,61	0,03	69,26	1040	HAIR
+Wilson-2015-173-43	43	HAND	manus	mano	hand	1	Noun	8	0,82	0,04	446,18	1277	HAND
+Wilson-2015-173-44	44	HE (THEY)	is	él	he (they)	4	Pronoun	11	2,45	0,05	1201,62		
```

Comment:
Maybe we can map this with `1211 HE`? I saw on the source that the meaning is for both SG. and PL. but in the Spanish glossing it only refers to the SG. The same happens with `#173 YOU (THOU)` that is linked to the singular CONCEPTICON_ID/GLOSS (and has also only the singular form in the Spanish glossing on the original source).